1_list_videos.py

```diff
@@ -1,39 +1,46 @@
-from selenium import webdriver
-from selenium.webdriver.chrome.options import Options
-from selenium.webdriver.support.ui import WebDriverWait
-from selenium.webdriver.support import expected_conditions as EC
-from selenium.webdriver.common.by import By
-import time
+import os
 import sqlite3
 
-courses = []
-with open ("courses.txt", "r") as f:
-	courses = [line.strip() for line in f.readlines()]
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
```

Comment:
Selenium's docs style it as EC, so I'd like to leave it that way
https://selenium-python.readthedocs.io/waits.html